File: LV1/1.6/leftbeehind/leftbeehind.py
while True:
    a, b = [int(i) for i in input().split(' ')]

    if a == 0 and b == 0:
        break

    if a + b == 13:
        print("Never speak again.")
    elif a == b:
        print("Undecided.")
    elif a > b:
        print("To the convention.")
    else:
        print("Left beehind.")

File: LV1/1.6/leftbeehind/leftbeehind.rs
use std::io::{self, BufRead};

fn main() {
    let stdin = io::stdin();

    for line in stdin.lock().lines().map(|l| l.unwrap()) {
        let num: Vec<i32> = line
            .split_whitespace()
            .map(|n| n.parse().unwrap())
            .collect();
        let a = num[0];
        let b = num[1];

        if a == 0 && b == 0 {
            break;
        }

        if a + b == 13 {
            println!("Never speak again.")
        } else if a == b {
            println!("Undecided.")
        } else if a > b {
            println!("To the convention.")
        } else {
            println!("Left beehind.")
        }
    }
}
